package/src/components/ExampleIOUPaymentForm/v1/ExampleIOUPaymentForm.js

```diff
@@ -0,0 +1,148 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Form } from "reacto-form";
+import { uniqueId } from "lodash";
+import { withComponents } from "@reactioncommerce/components-context";
+import { CustomPropTypes } from "../../../utils";
+
+/**
+ * Convert the form document to the object structure
+ * expected by `PaymentsCheckoutAction`
+ * @param {Object} Form object
+ * @returns {Object} Transformed object
+ */
+function buildResult({ amount, fullName }) {
+  let floatAmount = amount ? parseFloat(amount) : null;
+  if (isNaN(floatAmount)) floatAmount = null;
+
+  return {
+    amount: floatAmount,
+    data: { fullName },
+    displayName: fullName ? `IOU from ${fullName}` : null
+  };
+}
+
+class ExampleIOUPaymentForm extends Component {
+  static propTypes = {
+    /**
+     * You can provide a `className` prop that will be applied to the outermost DOM element
+     * rendered by this component. We do not recommend using this for styling purposes, but
+     * it can be useful as a selector in some situations.
+     */
+    className: PropTypes.string,
+    /**
+     * If you've set up a components context using
+     * [@reactioncommerce/components-context](https://github.com/reactioncommerce/components-context)
+     * (recommended), then this prop will come from there automatically. If you have not
+     * set up a components context or you want to override one of the components in a
+     * single spot, you can pass in the components prop directly.
+     */
+    components: PropTypes.shape({
+      /**
+       * Pass either the Reaction ErrorsBlock component or your own component that
+       * accepts compatible props.
+       */
+      ErrorsBlock: CustomPropTypes.component.isRequired,
+      /**
+       * Pass either the Reaction Field component or your own component that
+       * accepts compatible props.
+       */
+      Field: CustomPropTypes.component.isRequired,
+      /**
+       * Pass either the Reaction TextInput component or your own component that
+       * accepts compatible props.
+       */
+      TextInput: CustomPropTypes.component.isRequired
+    }),
+    /**
+     * Is the payment input being saved?
```

Comment:
Same comment as early, maybe add "currently" here to signify it's an ongoing action